content/reference/cli.md

```diff
@@ -1423,6 +1424,29 @@ Update the remote with local history, overwriting any conflicting history in the
 
 
 
+## `dolt query-diff`
+
+Calculates table diff between two queries.
+
+**Synopsis**
+
+```bash
+dolt query-diff [options] [<query1>] [<query2>]
+```
+
+**Description**
+
+Will execute two queries and compare the resulting table sets.
+
+**Arguments and options**
+
+`<query1>`: A SQL `SELECT` query to be executed. 
+
+`<query2>`: A SQL `SELECT` query to be executed.
+
+
```

Comment:
Add a note like: 

**Note**

Query diff is performed brute force and thus, will be slow for large result sets. The algorithm is super linear (n^2) on the size of the results sets. Over time, we will optimize this to use features of the storage engine to improve performance.